tests/test_fairworkflow.py

```diff
@@ -71,6 +71,9 @@ def test_construct_from_rdf_including_steps(self):
             assert step.uri in valid_step_uris
         workflow.validate()
 
+        # Check that workflow rdf passes plex shacl validation
+        assert workflow.shacl_validate()
```

Comment:
Please also add a test where this shacl validation fails.

Reply:
Done.